nextflow_schema.json

```diff
@@ -166,6 +118,107 @@
+            "properties": {
+                "custom_config_version": {
+                    "type": "string",
+                    "description": "Git commit id for Institutional configs.",
+                    "default": "master",
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog"
+                },
+                "custom_config_base": {
+                    "type": "string",
+                    "description": "Base directory for Institutional configs.",
+                    "default": "https://raw.githubusercontent.com/nf-core/configs/master",
+                    "hidden": true,
+                    "help_text": "If you're running offline, Nextflow will not be able to fetch the institutional config files from the internet. If you don't need them, then this is not a problem. If you do need them, you should download the files from the repo and tell Nextflow where to find them with this parameter.",
+                    "fa_icon": "fas fa-users-cog"
+                },
+                "config_profile_name": {
+                    "type": "string",
+                    "description": "Institutional config name.",
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog"
+                },
+                "config_profile_description": {
+                    "type": "string",
+                    "description": "Institutional config description.",
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog"
+                },
+                "config_profile_contact": {
+                    "type": "string",
+                    "description": "Institutional config contact information.",
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog"
+                },
+                "config_profile_url": {
+                    "type": "string",
+                    "description": "Institutional config URL link.",
+                    "hidden": true,
+                    "fa_icon": "fas fa-users-cog"
+                }
+            }
+        },
+        "bigstitcher_options": {
+            "title": "BigStitcher options",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "bigstitcher_work_dir": {
+                    "type": "string",
+                    "default": "work",
+                    "description": "Work directory"
+                },
+                "bigstitcher_distributed": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Flag to use a distributed spark cluster for BigStitcher"
+                },
+                "bigstitcher_spark_workers": {
```

Comment:
Do we need to prepend all these options with "bigstitcher"? This pipeline won't have any other Spark clusters, so it seems safe to say e.g. "spark_workers" and it's easier for the user to type on the CLI.

Reply:
I can remove the prefix